install_config/aggregate_logging.adoc

```diff
@@ -12,31 +12,33 @@ toc::[]
 
 == Overview
 
-OpenShift cluster administrators can aggregate and view logs for a range of
-OpenShift services using the EFK stack, which is a modified version of the
-https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. This
-topic shows you how to deploy the stack using a template.
+OpenShift cluster administrators can aggregate logs and
+enable users to view them for a range of OpenShift services
+using the EFK stack, which is a modified version of the
+https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK
+stack]. The EFK stack can be useful for viewing logs aggregated from
+hosts and applications, whether coming from multiple containers or even
+deleted pods.
```

Comment:
Trying to emphasize that this is not just an admin tool. Regular users can use it for their own logs, and it represents new features for doing that.